### ISSUE TITLE
Priority 1 accessibility issue: animations cannot be stopped

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6562]
**What**

There are animations present on the page, and while they have associated pause buttons, upon scrolling the page or navigating further via keyboard, the animations begin to play again and cannot be stopped permanently. This can be very distracting to some users.

<img width="424" alt="Screenshot 2021-05-27 at 16 44 11" src="https://user-images.githubusercontent.com/7315126/119856610-c58f1980-bf0a-11eb-9320-6a91bb4f2db3.png">

<img width="584" alt="Screenshot 2021-05-27 at 16 44 34" src="https://user-images.githubusercontent.com/7315126/119856673-d3dd3580-bf0a-11eb-8921-b0a8faf0482f.png">

**Where**

URL: https://wellcomecollection.org/articles/YACAExAAACEAqslr Page: Drug sharing in desperate times | Wellcome Collection
